sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/PackagingTests/DocGeneratorTest.cs

```diff
@@ -41,7 +41,7 @@ public void CheckNumberOfCrossReferences()
         {
             // These are the deprecated rules which reference their successors
             var crossReferenceCount = GetNumberOfCrossReferences(AnalyzerLanguage.CSharp);
-            crossReferenceCount.Should().Be(6);
+            crossReferenceCount.Should().Be(7);

```

Comment:
This test counts the cross-references between rules. We recently added a reference (e.g. `see` section in the description) in one of the IDisposable rules...